Mathlib/Analysis/Asymptotics/ExpGrowth.lean

```diff
@@ -118,22 +118,26 @@ lemma frequently_exp_le (h : a < expGrowthSup u) :
 lemma _root_.Frequently.expGrowthInf_le (h : ∃ᶠ n : ℕ in atTop, u n ≤ exp (a * n)) :
     expGrowthInf u ≤ a := by
   apply expGrowthInf_le_iff.2 fun c c_u ↦ h.mono fun n hn ↦ hn.trans ?_
-  exact exp_monotone (mul_le_mul_of_nonneg_right c_u.le n.cast_nonneg')
+  gcongr
+  apply Nat.cast_nonneg'
```

Comment:
Isn't this a sign that we should teach positivity about `Nat.cast_nonneg'`, so `gcongr` can use it?

Reply:
Yes, it is.